src/visualization/types/Graph/view.tsx

```diff
@@ -210,6 +223,7 @@ const XYPlot: FC<Props> = ({
         interpolation,
         position: properties.position,
         colors: colorHexes,
+        colorMapping,
```

Comment:
an alternative to passing in `colorMapping: null` when the flag is off is doing something like:

remove this line in the definition of `config`, and then below where `config` is defined, doing this:

```ts
if (isFlagEnabled('graphColorMapping')) {
  config.colorMapping = colorMapping
}
```

to make sure this is truly 'dark' in the sense that it makes no changes to production code if the flag is off.